### PR TITLE
Modularize into an installable app

### PR DESCRIPTION
**Pull request by jacobtylerwalls**
Make changes needed for installing this project as an "arches installed application" in 7.5.

Depends on #1268

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        "type": "git"
+    },
+    "version": "1.1.0",
+    "dependencies": {
+        "@uppy/aws-s3-multipart": "^2.4.1",
+        "@uppy/core": "^2.3.1",
+        "@uppy/dashboard": "^2.4.0",
+        "@uppy/drag-drop": "^2.1.2",
+        "@uppy/progress-bar": "^2.1.1",
+        "arches": "archesproject/arches#9304-cbyrd-arches-installed-packages",
```
Comment (jacobtylerwalls) [inline package.json]: TODO, fix when merged

File: package.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
+{
```
Comment (jacobtylerwalls) [inline package.json:1]: TODO: Moving this file is going to necessitate updates in the Docker config.

**Comment by jacobtylerwalls** [comment]
Most of the worfklows look fine, but init-workflow doesn't. Maybe the import of afs-settings is not working. Need to retest with/without #1286.

**Review by aarongundel — APPROVED** [review]
LGTM

**Comment by jacobtylerwalls** [comment]
Thanks for checking!

**Pull request merged**